jobs/image/cos_beta.env

```diff
@@ -0,0 +1,3 @@
+JENKINS_USE_GCI_VERSION=y
+JENKINS_GCI_HEAD_IMAGE_FAMILY=gci-59
```

Comment:
hummm, true... @abgworrall now it let me think that we should implement something like `--extract=gci-stable` and dereference it from some config file.